### PR TITLE
Clean headers from adown

### PR DESCRIPTION
**Pull request by brianlball**

### CODE REVIEW

File: src/EnergyPlus/DataAirSystems.hh [hunk 1]
```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
...
-
-        // Default Constructor
-        ConnectZoneComp()
-            : ZoneEqListNum(0), ZoneEqCompNum(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0), PlantLoopComp(0), FirstDemandSidePtr(0),
-              LastDemandSidePtr(0)
-        {
-        }
+        int ZoneEqListNum = 0;
+        int ZoneEqCompNum = 0;
+        int PlantLoopType = 0;
```
Comment (amirroth) [inline src/EnergyPlus/DataAirSystems.hh:252]: :pushpin: we just defined `struct ConnectionPoint` and now here is a logical instance of that `struct` and ... we're not using it.

File: src/EnergyPlus/DataAirSystems.hh [hunk 2]
```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
...
-        // Default Constructor
-        ConnectZoneSubComp()
-            : ZoneEqListNum(0), ZoneEqCompNum(0), ZoneEqSubCompNum(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0), PlantLoopComp(0),
-              FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int ZoneEqListNum = 0;
+        int ZoneEqCompNum = 0;
+        int ZoneEqSubCompNum = 0;
+        int PlantLoopType = 0;
```
Comment (amirroth) [inline src/EnergyPlus/DataAirSystems.hh:266]: :pushpin: and another instance.

File: src/EnergyPlus/DataAirSystems.hh [hunk 3]
```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
...
-        ConnectZoneSubSubComp()
-            : ZoneEqListNum(0), ZoneEqCompNum(0), ZoneEqSubCompNum(0), ZoneEqSubSubCompNum(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0),
-              PlantLoopComp(0), FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int ZoneEqListNum = 0;
+        int ZoneEqCompNum = 0;
+        int ZoneEqSubCompNum = 0;
+        int ZoneEqSubSubCompNum = 0;
+        int PlantLoopType = 0;
```
Comment (amirroth) [inline src/EnergyPlus/DataAirSystems.hh:281]: :pushpin:

File: src/EnergyPlus/DataAirSystems.hh [hunk 4]
```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
...
-        // Default Constructor
-        ConnectAirSysComp()
-            : AirLoopNum(0), AirLoopBranch(0), AirLoopComp(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0), PlantLoopComp(0),
-              FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int AirLoopNum = 0;
+        int AirLoopBranch = 0;
+        int AirLoopComp = 0;
+        int PlantLoopType = 0;
```
Comment (amirroth) [inline src/EnergyPlus/DataAirSystems.hh:295]: :pushpin:

File: src/EnergyPlus/DataAirSystems.hh [hunk 5]
```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
...
-        ConnectAirSysSubComp()
-            : AirLoopNum(0), AirLoopBranch(0), AirLoopComp(0), AirLoopSubComp(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0),
-              PlantLoopComp(0), FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int AirLoopNum = 0;
+        int AirLoopBranch = 0;
+        int AirLoopComp = 0;
+        int AirLoopSubComp = 0;
+        int PlantLoopType = 0;
```
Comment (amirroth) [inline src/EnergyPlus/DataAirSystems.hh:310]: :pushpin:

File: src/EnergyPlus/DataAirSystems.hh [hunk 6]
```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
...
-            : AirLoopNum(0), AirLoopBranch(0), AirLoopComp(0), AirLoopSubComp(0), AirLoopSubSubComp(0), PlantLoopType(0), PlantLoopNum(0),
-              PlantLoopBranch(0), PlantLoopComp(0), FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int AirLoopNum = 0;
+        int AirLoopBranch = 0;
+        int AirLoopComp = 0;
+        int AirLoopSubComp = 0;
+        int AirLoopSubSubComp = 0;
+        int PlantLoopType = 0;
```
Comment (amirroth) [inline src/EnergyPlus/DataAirSystems.hh:326]: :pushpin:. Also, each of these `struct` is a subtype of the previously defined `struct` with one added field. Just sayin`.

Comment (brianlball) [reply src/EnergyPlus/DataAirSystems.hh:326]: https://github.com/NREL/EnergyPlus/issues/9372

**Pull request merged**